src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -210,6 +212,23 @@ public void Configuration(IAppBuilder app)
             Configuration(app, new ConfigurationFactory(configurationProvider), null, null);
         }
 
+        private async Task<BasicSearchConfiguration> GetConfiguration()
+        {
+            try
+            {
+                return await _configFactory.Get<BasicSearchConfiguration>();
+            }
+            catch (KeyVaultClientException e) //when (e.Status == HttpStatusCode.Unauthorized || e.Status == HttpStatusCode.Forbidden) 
```

Comment:
Let's move this `when` to a comment in English. This just looks like forgotten code